### DISCUSSION TITLE
Major Tournament Format suggestion

### DISCUSSION THREAD

**Discussion by Nettogrof** [BattlesnakeOfficial/feedback#87] [General]
For any major tournament (like summer league or winter classic) the current goals are : 

1. Give a visibility to lower tier snakes. Seeing your snake during the tournament can be a great motivation, specially for new battlensaker
2. Limit the number of broadcasted games. Nobody want a tournament that last too long.
3. Having best snakes competition.

Currently those goals are reached by : 

1. Showing Gold , platinum and elite tier.
2. Limiting the number of snakes  (Top 16 in elite , top 32  Platinum and Gold top 32 snakes)
3. Having the top 16 elite tier

In my humble opinion the current format is great.

But I believe there's still some issue/negative points that could be improve : 

1. If a snake reach a new tier at the end of the season (example top gold reaching Platinum tier, on the last day), the snake won't appear during the tournament and no chance to win any prize.
2. Any snake that join the league late have less chance to be in the tournament even with the best snake. Even a perfect snake it took more than 5 days to reach elite, even more to be sure to be in the top 16.  Preventing also last week improvements to be relevant for the tournament.
3. Being in a tournament can be sometime determined by the last game of the league

Here my suggestion:
Doing a qualification tourney to fill the last place like in tennis.  Example Elite tournament could be the top 14 snakes are automatically in, and the qualification tourney will determine the last two spot.  Same for other tiers.
The qualification tourney could be offline or streamed without caster on the tournament day.

Number could be change  like it could also be Top 8 are automatically in and 8 qualification spot.

**Comment by bvanvugt** [comment]
Specifics aside, we really like this idea! 

Offline / non-shoutcasted qualifiers would give more Battlesnakes an opportunity to compete in league-based tournaments - especially considering that the barrier to more tournament games is just the tournament is getting way too long to broadcast live.

And since the community is growing at a quick pace, offline solutions make a lot of sense. 

That said, it's unlikely we're going to address this in Fall League 2021, more likely a 2022 Season fix.
